tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/CallStackDebugTest.java

```diff
@@ -83,7 +83,7 @@ public void stackFrameDebugTest() throws BallerinaTestException {
         // Stack frame representation test for strand creation with 'start' keyword.
         // Results of the strand is not assigned to any variable. In this case frame name is assigned to 'anonymous'.
         debugTestRunner.assertCallStack(frames[0], "sayHello", 45, "main.bal");
-        debugTestRunner.assertCallStack(frames[1], "start:anonymous", 10, "main.bal");
+        debugTestRunner.assertCallStack(frames[1], "start:anon", 10, "main.bal");
```

Comment:
Here, the default strand name is changed to `anon` if the future value name is not provided.